.github/workflows/publish-canary.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - lido-ui-v4-canary
+  workflow_dispatch:
```

Comment:
just mentioning: with this trigger you'd be able to deploy _any branch_ as a canary because checkout action would pull branch you selected when triggering the workflow

Reply:
same for other workflow in this PR